redis-snapshot-replicator/main.tf

```diff
@@ -275,18 +284,16 @@ resource "aws_lambda_function" "redis_copy_snapshot" {
 
   environment {
     variables = {
-      DB_INSTANCES  = join(",", var.db_instances)
       TARGET_BUCKET = aws_s3_bucket.bucket[0].id
     }
   }
 }
 
 resource "aws_sns_topic_subscription" "lambda_subscription" {
-  count         = var.enable ? 1 : 0
-  topic_arn     = var.redis_sns_topic_arn
-  protocol      = "lambda"
-  endpoint      = aws_lambda_function.redis_copy_snapshot[0].arn
-  filter_policy = jsonencode(map("Message", map("prefix", "Snapshot succeeded")))
+  count     = var.enable ? 1 : 0
+  topic_arn = var.redis_sns_topic_arn
+  protocol  = "lambda"
+  endpoint  = aws_lambda_function.redis_copy_snapshot[0].arn
```

Comment:
I think it's still a good idea to add the `filter_policy` here, no? This way the lambda function will only be triggered on snapshot events, which is the only thing we care for now.

Reply:
`filter_policy` works on message attributes only while ElasitCache sends the SNS message with no attributes. That's why in the first place I added a lambda that filters event and sends the proper attributes so the filter policy can work

Reply:
I see, that's too bad